collections-csharp/ConcurrentQueueInCSharp/ConcurrentQueueInCSharp.Console/Consumer.cs

```diff
@@ -0,0 +1,28 @@
+﻿using ConcurrentQueueInCSharp.Models;
+
+namespace ConcurrentQueueInCSharp
+{
+    public class Consumer
+    {
+        private readonly OrderMessageBus _messageBus;
+
+        public Consumer(OrderMessageBus messageBus)
+        {
+            _messageBus = messageBus;
+        }
+
+        public Task Process()
+        {
+            return Task.Factory.StartNew(() =>
+            {
+                Order order;
+                string processId = Guid.NewGuid().ToString();
```

Comment:
This can be var, but better still, we can just move this code (minus the `ToString` into the interpolated string

Reply:
Good point.